Makefile

```diff
@@ -757,6 +757,13 @@ pkgs: RESCAN_DEPS=
 pkgs: build-tools $(PKGS)
 	@echo Done building packages
 
+compose-external-boot-image:
```

Comment:
You don't need to use a PHONY target here and generate the build.yml all the time. You can use build.yml as the target and build.yml.in as source, like the following:

```make
pkg/external-boot-image/build.yml: pkg/external-boot-image/build.yml.in
```

Also, compose-external-boot-image is not a dependency for any package, so if the build.yml file is not present at pkg/external-boot-image folder, it will fail if I hit manually `make pkg/external-boot-image` before run `make compose-external-boot-image`